src/engine/strat_engine/tests/loopbacked.rs

```diff
@@ -43,17 +44,19 @@ impl LoopTestDev {
     pub fn new(lc: &LoopControl, path: &Path, size: Option<Sectors>) -> LoopTestDev {
         let size = size.unwrap_or(Bytes(IEC::Gi).sectors());
 
-        let mut f = OpenOptions::new()
+        let f = OpenOptions::new()
             .read(true)
             .write(true)
             .create(true)
             .open(&path)
             .unwrap();
 
-        // the proper way to do this is fallocate, but nix doesn't implement yet.
-        // TODO: see https://github.com/nix-rust/nix/issues/596
-        f.seek(SeekFrom::Start(*size.bytes())).unwrap();
-        f.write(&[0]).unwrap();
+        fallocate(
+            f.as_raw_fd(),
+            FallocateFlags::empty(),
+            0,
+            *size.bytes() as i64,
```

Comment:
Any objection to casting to ```nix::libc::off_t``` instead, since that is the actual type of the parameter the method requires?